library/src/main/java/com/pokegoapi/api/inventory/Inventories.java

```diff
@@ -33,13 +33,13 @@
 import com.pokegoapi.exceptions.LoginFailedException;
 import com.pokegoapi.exceptions.RemoteServerException;
 import com.pokegoapi.main.ServerRequest;
+import com.pokegoapi.api.player.Stats;
 
 import lombok.Getter;
 
 import java.util.ArrayList;
 import java.util.List;
 
-
 public class Inventories {
```

Comment:
I know, I just told you that I haven't changed it in any way, I only "told" to the Inventory class to use a different Stats class with the import of "com.pokegoapi.api.player.Stats".